neural_speed/models/qwen/qwen_utils.cpp

```diff
@@ -150,6 +163,34 @@ void QWEN::load(model_context* ctx, model_progress_callback progress_callback, v
       layer.ffn[1] = ml->get_tensor(layers_i + ".mlp.w2.weight", {n_embd, n_ff}, backend);
       layer.ffn[2] = ml->get_tensor(layers_i + ".mlp.c_proj.weight", {n_ff, n_embd}, backend);
     }
+  } else {
```

Comment:
There are so many if-elses, can we split those if-else and add comments to make it clear whether it is qwen1, qwen2, gguf or bin?

Reply:
done